jellyfin_kodi/objects/kodi/movies.py

```diff
@@ -158,7 +158,7 @@ def omega_migration(self):
 
         # Sets all existing movies without a version to standard version
         for entry in self.cursor.fetchall():
-            self.add_videoversion(entry[0], entry[1], "movie", "0", 40400)
+            self.add_videoversion(entry[0], entry[1], "movie", "1", 40400)
```

Comment:
Notes for later:
Needs separate migration step for Piers, see earlier comment.
May or may not actually need to do this migration, as looking at the Kodi source, the migration is supposed to get applied on upgrade.

Probably need a check for db version and two different queries for adding instead.

Reply:
>May or may not actually need to do this migration, as looking at the Kodi source, the migration is supposed to get applied on upgrade.

I suspect the reason for including the migration in the addon is to repair libraries that have been broken by the addon. The only other way for users to recover from having the current version of this addon running on Piers would be to delete their movie libraries entirely and sync from scratch, but users won't know to do that and so they will just have missing library items, since the upstream Kodi migration has already been run prio to the addon syncing broken items.

If this addon was updated before anyone ran it on Piers, then this wouldn't be an issue, but that's not where we are now.